### ISSUE TITLE
NER in Chinese context 

### ISSUE THREAD

**Issue by Jaaaaaay** [clab/stack-lstm-ner#18]
Thank your guidance,  my bash script is like your commended code, "sudo ./lstm-parse -T data/huabi/conll_train_data.parser -d data/huabi/dev.parser --hidden_dim 100 --lstm_input_dim 100 -w conll_train_data.vectors.vec --pretrained_dim 100 --rel_dim 20 --action_dim 20 --input_dim 100 -t -S -D 0.3 > logNERYesCharNoPosYesEmbeddingsD0.3.txt &". the data/huabi/conll_train_data.parser is training dataset and the conll_train_data.vectors.vec is pre-trained word embedding, The error information is:
"REDUCE(DET_RESULT_POS) :: 18
REDUCE(BASE_TEMP) :: 19
Number of words: 9406
Number of UTF8 chars: 2084
Training started.
NUMBER OF TRAINING SENTENCES: 4074
**SHUFFLE

[1]  + 22935 segmentation fault   -T  -d  --hidden_dim 100 --lstm_input_dim 100 -w  --pretrained_dim 100  20 "

Thanks you patience!

**Comment by miguelballesteros** [comment]
It looks like that your data is not correctly formatted.
can we see the contents of logNERYesCharNoPosYesEmbeddingsD0.3.txt?

did you follow all the steps as in the readme?

thanks,
Miguel.

**Comment by Jaaaaaay** [comment]
Thank you, I solved this problem, you are right, my data is not correctly formatted, there are a double empty lines in the corpus, so the segmentation fault occurred.
Your programmer is great, it help me a lot, thank you again!

**Comment by miguelballesteros** [comment]
Thanks!
